admin/admin-service/src/test/java/io/mosip/admin/controller/test/BulkDataUploadControllerTest.java

```diff
@@ -86,6 +96,23 @@ public void setUp() throws Exception {
 		mockMvc.perform(MockMvcRequestBuilders.multipart("/bulkupload").file(gender).param("tableName","gender").param("operation","insert").param("category","masterdata"));
 
 	}
+	
+	@SuppressWarnings("unchecked")
+	@Test
+	@WithUserDetails("global-admin")
+	public void t012checkCountUpload() throws Exception {
+		String content="code,genderName,langCode,isActive\r\n" +
+				"MLIO,Test,eng,FALSE\r\n" +
+				"AABA,AAA,ara,TRUE\r\n" +
+				"BBCB,BBB,eng,TRUE";
+		MockMultipartFile gender = new MockMultipartFile("files", "gender.csv", "multipart/form-data", content.getBytes());
+		MvcResult response = mockMvc.perform(MockMvcRequestBuilders.multipart("/bulkupload").file(gender).param("tableName","gender").param("operation","insert").param("category","masterdata")).andReturn();
+		String transaction = response.getResponse().getContentAsString();
+		JSONParser parser = new JSONParser();
+		JSONObject json = (JSONObject) parser.parse(transaction);
+		JSONObject jsonResponse = (JSONObject) parser.parse(json.get("response").toString());
+		transactionId = jsonResponse.get("transcationId").toString();
```

Comment:
what are we asserting here ?

Reply:
Actually, I'm trying to add the csv data here and checking the response count in the last test class by saving the transactionId, So I didn't add any assertion. Now I added an assertion according to other test classes